data/aliases.ts

```diff
@@ -2509,35 +2605,63 @@ export const CompoundWordNames = [
 	'Zap-dos',
 
 	// moves
+	'10,000,000 Volt Thunder-bolt',
 	'Accele-rock',
 	'Acid Down-pour',
+	'Acro-batics',
```

Comment:
Hm. Are "bat" or "abat" or "ab" ever used as search terms? I don't know if this one makes sense.